libtxpool/TxPool.cpp

```diff
@@ -215,6 +224,14 @@ bool TxPool::isSealerOrObserver()
 ImportResult TxPool::import(Transaction::Ptr _tx, IfDropped)
 {
     _tx->setImportTime(u256(utcTime()));
+    auto memoryUsed = m_usedMemorySize + _tx->capacity();

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'memoryUsed' is assigned a value that is never used.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5432284)